test/core/iomgr/load_file_test.cc

```diff
@@ -56,6 +57,11 @@ static void test_load_empty_file(void) {
   GPR_ASSERT(GRPC_SLICE_LENGTH(slice_with_null_term) == 1);
   GPR_ASSERT(GRPC_SLICE_START_PTR(slice_with_null_term)[0] == 0);
 
+  time_t ts = gpr_last_modified_timestamp(tmp_name);
```

Comment:
The timestamp tests should be in their own file, since they're testing a separate module.